### PR TITLE
Fixed GcPlanetData

### PR DESCRIPTION
**Pull request by TheBob12**

### CODE REVIEW

**Comment by TheBob12** [comment]
Sorry about the mess wwith GcBuildingDensityLevels... it looked fine but changed with commit lol

File: libMBIN/Source/Models/Structs/GcPlanetData.cs [hunk 1]
```diff
@@ -2,66 +2,59 @@
...
+	[NMS(Size = 0x3BB0)]
+	public class GcPlanetData : NMSTemplate
+	{
         [NMS(Size = 0x80)]
-        /* 0x0000 */ public string Name;
-        /* 0x0080 */ public GcPlanetLife Life;
-        /* 0x0084 */ public GcPlanetLife CreatureLife;
-        /* 0x0088 */ public GcPlanetHazardData Hazard;
-
+		/* 0x0000 */ public string name;
```
Comment (monkeyman192) [inline libMBIN/Source/Models/Structs/GcPlanetData.cs]: really?

Comment (TheBob12) [reply libMBIN/Source/Models/Structs/GcPlanetData.cs]: IDK how that happened lol
... fixed

File: libMBIN/Source/Models/Structs/GcPlanetData.cs [hunk 2]
```diff
@@ -2,66 +2,59 @@
...
-        [NMS(Size = 2, Ignore = true)]
-        public byte[] Padding34D2;
-
-        /* 0x34D4 */ public GcPlanetSentinelData SentinelData;
-        /* 0x34DC */ public GcPlanetTradingData TradingData;
-        /* 0x34E4 */ public GcPlayerConflictData ConflictData;
-        [NMS(Size = 0x8, Ignore = true)]
-        public byte[] EndPadding;
-    }
+		/* 0x1CA0 */ public string TerrainFile;
```
[the remark this answers is not in the document]
Comment (TheBob12) [reply libMBIN/Source/Models/Structs/GcPlanetData.cs]: I'll fix it when I get home, today.

**Pull request merged**